rest-api-spec/src/main/resources/rest-api-spec/api/connector.update_api_key_id.json

```diff
@@ -0,0 +1,38 @@
+{
+  "connector.update_api_key_id": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/update-connector-api-key-id-api-key-secret-api.html",
```

Comment:
Should we stub it with non-404 url? Maybe just use `https://www.elastic.co/guide/en/elasticsearch/reference/master/connector-apis.html` and when you write the doc just replace it here?

Reply:
Replaced, sorry I thought we could put expected docs here even if they don't exist yet